assignment/tool/src/webapp/vm/assignment/chef_assignments_list_assignments.vm

```diff
@@ -156,6 +156,24 @@
 							#end
 						</a>
 					</th>
+					#if ($!isTimesheet)
+						<th id="timeSheet">
+							<a href="javascript:void(0)" onclick="location='#toolLinkParam("$action" "doSort" "criteria=timesheet")'; return false;"  title="$tlang.getString("gen.sorbytime")">
+								#if ($!allowGradeSubmission)
+									$tlang.getString("gen.timesheet.instructor")
+								#else
+									$tlang.getString("gen.timesheet.student")
+								#end
+								#if ($sortedBy.equals("timesheet"))
+									#if ($sortedAsc.equals("true"))
+										<img id="sortascendingtimesheet" src = "#imageLink("sakai/sortascending.gif")" border="0" alt="$tlang.getString("gen.sorasc")" />
```

Comment:
How do these images look in dark theme? Could we switch to font awesome icons?

Reply:
Done